README.md

```diff
@@ -420,6 +423,16 @@ Should output something like:
 {"jsonrpc":"2.0","result":"0x004ec07d2329997267ec62b4166639513386f32e","id":0}
 ```
 
+### Option 2: run Kovan node
```

Comment:
wouldn't it better just to right about kovan hear to avoid overloading the user?
and link to the other page where details about deploying on devchain are explained